### PR TITLE
Add hamburger menu for mobile version

### PR DESCRIPTION
**Pull request by mullenlucas**
## Main additions:

- Fix html `id` and `class` structure to fit the upcoming javascript code for manipulating the hamburger menu
- Add css styles to the hamburger menu
- Add `X` icon to close the menu
- Add javascript code in scripts.js to manage the behavior of the hamburger menu
- Add javascript code to show the hamburger menu items when resizing from mobile (hidden) to desktop (shown)

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const hambIcon = document.getElementById('hamburgerIcon');
+const closeIcon = document.getElementById('closeIcon');
```
Comment (roshan-bajgain) [inline js/script.js:2]: - [ ] I would suggest you make your `X` buttons to match the figma design
### your design
your

### Figma design
figma

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+x.onchange = (e) => {
+  const menuItems = document.getElementById('myMenuItems');
+  if (e.matches) {
+    menuItems.style.display = 'flex';
+    menuItems.style.visibility = 'visible';
+  } else {
+    menuItems.style.display = 'none';
+    menuItems.style.visibility = 'hidden';
+  }
+};
```
Comment (roshan-bajgain) [inline js/script.js:59]: - [ ] If you see the design then the background is totally white kindly fix it  to make it similar with figma design
### Your design
your desing
### Figma design
fffff

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @mullenlucas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/script.js (hunk 1 above)
Comment (mullenlucas) [reply js/script.js:2]: Updated to font-awesome's latest version (6.1.1), and adapted the icon's styling

File: js/script.js (hunk 2 above)
Comment (Munsa1) [reply js/script.js:59]: - [ ] As requested by the previous revieweer kindly match the Figma design as this is part of the requirements

Your design
image

Figma design
image

- Kindly remove the welcome text on the mobile menu, and ensure that the white background covers the fulllenght of the mobile view when you click the menu button as this is also part of the requirements.
- Check out my portfolio for ideas as I did the same one as well [Portfolio](https://github.com/Munsa1/Mobile-Version)

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @mullenlucas @RodChak, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — APPROVED** [review]
Hi @mullenlucas, 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**